projects/README.md

```diff
@@ -0,0 +1,101 @@
+# Group project
+
+## Objectives
+
+The objective of project work is to apply the knowledge gained in the course, in a group setting, to a selected (open) information retrieval problem.
+Specifically, groups first needed to establish a reasonable baseline, then develop one or multiple advanced methods aiming to improve over the baseline. Using a standard test collection, they had to experimentally compare the baseline and advanced solutions.
+
+## Projects
+
+The following projects were available to choose from:
+
+  * **[MS Marco Document re-ranking](https://microsoft.github.io/msmarco/)**
+    - Given a corpus of passages generate a ranked list of the top 1000 passages sorted by relevance. This task has also run as part of the TREC 2019 and 2020 Deep Learning track.
+    - Document collection: MS MARCO passage corpus (8.8M passages, 22GB)
+    - Resources
+      - [Task GitHub repo](https://github.com/microsoft/MSMARCO-Document-Ranking)
+      - [Relevance judgments for evaluation topics](https://trec.nist.gov/data/deep/2019qrels-docs.txt)
+      - [TREC 2019 Deep Learning track overview](https://arxiv.org/abs/2003.07820)
+    - [Discussion group (on Canvas)](https://stavanger.instructure.com/courses/8838/discussion_topics/120134)
+  * **[TREC 2020 Conversational Assistance](http://www.treccast.ai/)**
+    - Given a series of conversational utterances, identify relevant passages for each user utterance that satisfies the user's information need.
+    - Document collection: MARCO Ranking passages and Wikipedia (TREC CAR)
+    - Resources
+      - [Task GitHub repo](https://github.com/daltonj/treccastweb/tree/master/2020)
+      - [TREC 2020 Conversational Assistance track overview](https://trec.nist.gov/pubs/trec29/papers/OVERVIEW.C.pdf)
+      - [TREC 2020 Conversational Assistance judgments for evaluation topics](https://trec.nist.gov/data/cast/2020qrels.txt)
+    - [Discussion group (on Canvas)](https://stavanger.instructure.com/courses/8838/discussion_topics/120142)
+  * **[Semantic Answer Type Prediction 2020](https://smart-task.github.io/2020/)**
+    - Given a question in natural language, the task is to predict type of the answer using a set of candidates from a target ontology (DBpedia).
+    - Document collection: DBpedia ([dump 2016-10](http://downloads.dbpedia.org/2016-10/)) and/or Wikidata ([dump](https://dumps.wikimedia.org/wikidatawiki/entities/))
+    - Resources
+      - [Task GitHub repo](https://github.com/smart-task/smart-dataset)
+      - Papers: [Balog & Neumayer (2012)](https://krisztianbalog.com/files/cikm2012-querytypes.pdf), [Garigliotti et al. (2017)](https://krisztianbalog.com/files/sigir2017-qt.pdf), [Setty & Balog (2020)](https://krisztianbalog.com/files/iswc2020-smart.pdf)
+    - [Discussion group (on Canvas)](https://stavanger.instructure.com/courses/8838/discussion_topics/120145)
+
+
```

Comment:
Add `## Instructions`, and move all headers below a level down, i.e., `##` => `###`, `###` => `####`, etc.